CHANGELOG.md

```diff
@@ -1,5 +1,99 @@
 # Change Log
 
+## [1.0.0](https://github.com/stellargraph/stellargraph/tree/v1.0.0)
+
+[Full Changelog](https://github.com/stellargraph/stellargraph/compare/v0.11.0...v1.0.0)
+
+This 1.0 release of StellarGraph is the culmination of three years of active research and engineering to deliver an open-source, user-friendly library for machine learning (ML) on graphs and networks.
+
+Jump in to this release, with the new demos and examples:
+
+- [More helpful indexing and guidance in demo READMEs](demos/)
+- [Loading from Neo4j][neo4j-1.0.0]
+- [More explanatory Node2Vec link prediction][n2v-lp-1.0.0]
+- [Unsupervised `GraphSAGE` and `HinSAGE` via `DeepGraphInfomax`][dgi-1.0.0]
+- Graph classification [with `GCNSupervisedGraphClassification`][gc-gcn-1.0.0] and [with `DeepGraphCNN`][gc-dgcnn-1.0.0]
+- [Time series prediction using spatial information, using `GraphConvolutionLSTM`][gcn-lstm-1.0.0] (experimental)
+
+[neo4j-1.0.0]: demos/basics/loading-saving-neo4j.ipynb
```

Comment:
I think all of these notebooks will need to link to readthedocs?

Reply:
Yep I'm going to switch the links now that that one has landed